docs/source/tutorial/production.md

```diff
@@ -14,6 +14,8 @@ Studio's core features are free for everyone. All of the features in this tutori
 
 ## Create an Apollo account
 
+> Skip this if you already created an account to use the Apollo Studio Explorer.
+
 Visit [studio.apollographql.com](https://studio.apollographql.com) and click **Create an account**. You can sign up either with your GitHub account or by setting a username and password.
```

Comment:
I wonder if we should point everyone to https://studio.apollographql.com/dev for signup here? That route has a streamlined sign up form, compared to the regular one which will have you make an account, invite people, and choose if you want to start a trial.

Reply:
I'm going to leave this as-is for this PR because it impacts directions for the remainder of the chapter, but we can revisit afterward